src/lib/stitching/gravity/__tests__/stitching.test.ts

```diff
@@ -707,4 +707,55 @@ describe("gravity/stitching", () => {
       expect(formattedDescription).toEqual("**Bold Type**")
     })
   })
+
+  describe("#artworksCountMessage", () => {
+    it("prefers for-sale artworks count", async () => {
+      const { resolvers } = await getGravityStitchedSchema()
+      const { artworksCountMessage } = resolvers.ArtistSeries
+      const value = await artworksCountMessage.resolve({
+        forSaleArtworksCount: 20,
+        artworksCount: 10,
+      })
+      expect(value).toEqual("20 available")
+    })
+
+    it("falls back to general artworks count", async () => {
+      const { resolvers } = await getGravityStitchedSchema()
+      const { artworksCountMessage } = resolvers.ArtistSeries
+      const value = await artworksCountMessage.resolve({
+        forSaleArtworksCount: 0,
+        artworksCount: 10,
+      })
+      expect(value).toEqual("10 works")
+    })
+
+    it("pluralizes correctly", async () => {
+      const { resolvers } = await getGravityStitchedSchema()
+      const { artworksCountMessage } = resolvers.ArtistSeries
+
+      const one = await artworksCountMessage.resolve({
+        forSaleArtworksCount: 0,
+        artworksCount: 1,
+      })
+      const many = await artworksCountMessage.resolve({
+        forSaleArtworksCount: 0,
+        artworksCount: 42,
+      })
+
+      expect(one).toEqual("1 work")
+      expect(many).toEqual("42 works")
+    })
+
+    it("returns null when there are no artworks at all", async () => {
+      const { resolvers } = await getGravityStitchedSchema()
+      const { artworksCountMessage } = resolvers.ArtistSeries
+
+      const value = await artworksCountMessage.resolve({
+        forSaleArtworksCount: 0,
+        artworksCount: 0,
+      })
+
+      expect(value).toEqual(null)
+    })
```

Comment:
This is not specified in any design but I figured returning a falsy value and not displaying a count at all was better than "0 works" — since that's what we were trying to avoid in the first place.

Reply:
Ohh interesting. I probably would expect this to say `0 works`! But if that would cause us to then do some logic/checks on the presence of the "0 works" string, then this makes sense.

Reply:
Cool, so I've put in a question to Nicole about whether we should show a "0 works" message or nothing at all for this edgy edge case — and will let that guide the decision 😎.

Reply:
Sounds good!

Reply:
Confirmed that we want to show "0 works" — commit incoming

Reply:
Fixed in https://github.com/artsy/metaphysics/pull/2617/commits/668c3733457c6075c765deeed9c6ff22754418ca#diff-e9a6ef1ded9b9a8e03c9320518a09a23R749